spec/rubocop/cop/style/if_unless_modifier_of_if_unless_spec.rb

```diff
@@ -0,0 +1,69 @@
+# encoding: utf-8
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::IfUnlessModifierOfIfUnless do
+  include StatementModifierHelper
+
+  subject(:cop) { described_class.new }
+
+  it 'provides a good error message' do
+    source = 'condition ? then_part : else_part unless external_condition'
```

Comment:
Yes, the error message is not dependant on the context, so I skipped it by lack of a meaningful context name. Any suggestions?